packages/collector/test/tracing/cloud/aws-sdk/v3/sqs/test_definition.js

```diff
@@ -23,7 +23,7 @@ const {
 const ProcessControls = require('../../../../../test_util/ProcessControls');
 const globalAgent = require('../../../../../globalAgent');
 const { verifyHttpRootEntry, verifyHttpExit } = require('@instana/core/test/test_util/common_verifications');
-const defaultPrefix = 'https://sqs.us-east-2.amazonaws.com/410797082306/';
+const defaultPrefix = 'https://sqs.us-east-2.amazonaws.com/767398002385/';
```

Comment:
issue: Please add the aws id to a centralized place for collector & aws-lambda pkg.